legacy/toml-config/src/main/java/io/zeebe/legacy/tomlconfig/LegacyConfigurationSupport.java

```diff
@@ -0,0 +1,257 @@
+    VALUE_CONVERTERS_GATEWAY.put(
+        "ZEEBE_GATEWAY_CONTACT_POINT",
+        value -> value.contains(":") ? value : value + ":" + DEFAULT_CONTACT_POINT_PORT);
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_CLUSTER_NAME", new Replacement("zeebe-gateway.cluster.clusterName"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_CLUSTER_MEMBER_ID", new Replacement("zeebe-gateway.cluster.memberId"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_CLUSTER_HOST", new Replacement("zeebe-gateway.cluster.host"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_CLUSTER_PORT", new Replacement("zeebe-gateway.cluster.port"));
+
+    // zeebe-gateway.monitoring
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_MONITORING_ENABLED", new Replacement("zeebe-gateway.monitoring.enadbled"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_MONITORING_HOST", new Replacement("zeebe-gateway.monitoring.host"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_MONITORING_PORT", new Replacement("zeebe-gateway.monitoring.port"));
+
+    // zeebe-gateway.threads
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_MANAGEMENT_THREADS",
+        new Replacement("zeebe-gateway.threads.managementThreads"));
+
+    // zeebe-gateway.security
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_SECURITY_ENABLED", new Replacement("zeebe-gateway.security.enabled"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_CERTIFICATE_PATH",
+        new Replacement("zeebe-gateway.security.certificateChainPath"));
+    MAPPING_GATEWAY.put(
+        "ZEEBE_GATEWAY_PRIVATE_KEY_PATH", new Replacement("zeebe-gateway.security.privateKeyPath"));
+
+    // zeebe-broker ----------------------------------------
+    MAPPING_BROKER.put("ZEEBE_STEP_TIMEOUT", new Replacement("zeebe-broker.stepTimeout"));
+
+    // zeebe-broker.cluster
+    MAPPING_BROKER.put("ZEEBE_NODE_ID", new Replacement("zeebe-broker.cluster.nodeId"));
+    MAPPING_BROKER.put(
+        "ZEEBE_CONTACT_POINTS", new Replacement("zeebe-broker.cluster.initialContactPoints"));
+    MAPPING_BROKER.put(
+        "ZEEBE_PARTITIONS_COUNT", new Replacement("zeebe-broker.cluster.partitionsCount"));
+    MAPPING_BROKER.put(
+        "ZEEBE_REPLICATION_FACTOR", new Replacement("zeebe-broker.cluster.replicationFactor"));
+    MAPPING_BROKER.put("ZEEBE_CLUSTER_SIZE", new Replacement("zeebe-broker.cluster.clusterSize"));
+    MAPPING_BROKER.put("ZEEBE_CLUSTER_NAME", new Replacement("zeebe-broker.cluster.clusterName"));
+
+    // zeebe-broker.data
+    MAPPING_BROKER.put("ZEEBE_DIRECTORIES", new Replacement("zeebe-broker.data.directories"));
+
+    // zeebe-broker.gateway
+    MAPPING_BROKER.put("ZEEBE_EMBED_GATEWAY", new Replacement("zeebe-broker.gateway.enable"));
+
+    /* this essentially copies all entries from the gateway mapping table, but modifies
+     * the new environment variables from "zeebe-gateway.*" to "zeebe-broker.gateway.*"
+     */
+    MAPPING_GATEWAY
+        .entrySet()
+        .forEach(
```

Comment:
```suggestion
    MAPPING_GATEWAY
        .forEach(
```

I think it can be called directly now

Reply:
done